device-types/TP-Link/TL-SG105.yaml

```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: TP-Link
+model: TL-SG105
+slug: tp-link-tl-sg105
+part_number: TL-SG105
+comments: '[TP-Link TL-SG105](https://www.tp-link.com/us/business-networking/soho-switch-unmanaged/tl-sg105/#specifications)'
+is_full_depth: false
+u_height: 0
```

Comment:
As you have added the images you should add the following

front_image: true
rear_image: true

Also add the front image option to the TL-SG105E device